app/assets/javascripts/initializePage.js

```diff
@@ -0,0 +1,82 @@
+/*
+  global initializeLocalStorageRender, initializeStylesheetAppend, initializeBodyData,
+  initializeAllChatButtons, initializeAllTagEditButtons, initializeUserFollowButts,
+  initializeBaseTracking, initializeTouchDevice, initializeCommentsPage,
+  initializeArticleDate, initializeArticleReactions, initNotifications,
+  initializeCommentDate, initializeCommentDropdown, initializeSettings,
+  initializeFooterMod, initializeCommentPreview, initializeAdditionalContentBoxes,
+  initializeTimeFixer, initializeDashboardSort, initializePWAFunctionality,
+  initializeEllipsisMenu, initializeArchivedPostFilter, initializeCreditsPage,
+  initializeUserProfilePage, initializePodcastPlayback, initializeDrawerSliders,
+  initializeHeroBannerClose, initializeOnboardingTaskCard, initScrolling,
+  nextPage:writable, fetching:writable, done:writable, adClicked:writable
```

Comment:
LOL.

I hope we can move to something like Stimulus to reorganize `app/assets/javascripts` at some point without having to go full "componenty" with Preact.